proposals/xxxx-mesh-nodes.md

```diff
@@ -0,0 +1,329 @@
+
+##### Mesh Node Output System Values
+
+Like mesh shaders, `vertices` and `primitives` outputs require semantics on
+fields to produce corresponding output and primitive signature elements.
+Outputs also support the same system values supported by mesh shaders, listed
+below.  See links below for additional details relevant to mesh shaders for
+`SV_PrimitiveID` and `SV_CullPrimitive`.
+
+| system value semantic       | type     | required? | location | notes |
+|-----------------------------|----------|-----------|----------|-------------|
+| `SV_Position`               | `float4`     | Y | `vertices`   |  |
+| `SV_RenderTargetArrayIndex` | `uint`       | N | `primitives` |  |
+| `SV_ViewPortArrayIndex`     | `uint`       | N | `primitives` |  |
+| `SV_ClipDistance`           | `float<1-4>` | N | `vertices`   |  |
+| `SV_CullDistance`           | `float<1-4>` | N | `vertices`   |  |
+| `SV_ShadingRate`            | `uint`       | N | `primitives` |  |
+| `SV_PrimitiveID`            | `uint`       | N | `primitives` | See [mesh shader spec](https://github.com/microsoft/DirectX-Specs/blob/master/d3d/MeshShader.md#sv_primitiveid-in-the-pixel-shader) |
+| `SV_CullPrimitive`          | `uint`       | N | `primitives` | See [mesh shader spec](https://github.com/microsoft/DirectX-Specs/blob/master/d3d/MeshShader.md#sv_cullprimitive) |
+
+#### SetMeshOutputCounts
+
+> Excerpts from [SetMeshOutputCounts](https://github.com/microsoft/DirectX-Specs/blob/master/d3d/MeshShader.md#setmeshoutputcounts) in the mesh shader spec.
+
+```hlsl
+void SetMeshOutputCounts(
+    uint numVertices,
+    uint numPrimitives);
+```
+
+The intrinsic function `SetMeshOutputCounts` must be called before writing to
+any of the output arrays.
+
+Some restrictions on the function use and interactions with output arrays follow.
+1. This function can only be called once per shader.
+2. This call must occur before any writes to any of the shared output arrays. The validator will verify this is the case.
+3. If the compiler can prove that this function is not called, then the threadgroup doesn't have any output. If the shader writes to any of the shared output arrays, compilation and shader validation will fail. If the shader does not call any of these functions, the compiler will issue a warning, and no rasterization work will be issued.
+4. Only the input values from the first active thread are used.
+5. This call must dominate all writes to shared output arrays. In other words, there must not be any execution path that even appears to reach any writes to any output array without first having executed this call.
+
+See [SetMeshOutputCounts](https://github.com/microsoft/DirectX-Specs/blob/master/d3d/MeshShader.md#setmeshoutputcounts)
+in the mesh shader spec for specific details, restrictions, and examples.
+
+### Interchange Format Additions
+
+To the `DXIL::NodeLaunchType` enum, add `Mesh` (4).
```

Comment:
Mentioning how this is represented in the metatdata tag explicitly would make it clear how this applies to the interchange format